content/help/faq/security/istio-to-not-istio.md

```diff
@@ -1,5 +1,8 @@
 ---
-title: Can a service with Istio Auth enabled communicate with a service without Istio?
+title: Can a service with Istio mutual TLS (mTLS) enabled talk to a service without Istio,
+and what about the other way around?
 weight: 20
 ---
-This is not supported currently, but will be in the near future.
+Starting from 0.8 release, a service with Istio mTLS enabled can talk to a service without Istio. Mutual TLS is enabled via [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and this only specifies the service behavior as a server, not client, which means a mTLS enabled service will still send http traffic (not mTLS) to others unless you explicitly specify it with [destination rule]({{home}}/docs/reference/config/istio.networking.v1alpha3.html#DestinationRule).
```

Comment:
Replace uses of mTLS with "mutual TLS", replace "cert" with "certificate"

Reply:
Done.